packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -250,4 +255,33 @@ void main() {
       }),
     ]);
   });
+
+  testWidgets('EditableText identifies as text field (w/ focus) in semantics', (WidgetTester tester) async {
+    final SemanticsTester semantics = new SemanticsTester(tester);
+
+    await tester.pumpWidget(
+      new Directionality(
+        textDirection: TextDirection.ltr,
+        child: new FocusScope(
+          node: focusScopeNode,
+          autofocus: true,
+          child: new EditableText(
+            controller: controller,
+            focusNode: focusNode,
+            style: textStyle,
+            cursorColor: cursorColor,
+          ),
+        ),
+      ),
+    );
+
+    expect(semantics, includesNodeWith(flags: <SemanticsFlags>[SemanticsFlags.isTextField]));
+
+    await tester.tap(find.byType(EditableText));
+    await tester.idle();
+    await tester.pump();
+
+    expect(semantics, includesNodeWith(flags: <SemanticsFlags>[SemanticsFlags.isTextField, SemanticsFlags.isFocused]));
+
+  });
```

Comment:
Should we test the non focused case as well?

Reply:
Isn't that covered by line 278?

Reply:
oops